src/apiary.apib

```diff
@@ -1544,6 +1544,61 @@ level `conversions` array.
                 + conversion (Conversion)
                     + conversion_type: 3 (number)
 
+# Group Alert
+
+Alerts are used across application to notify users (when subscribed) of events happening in platform.
+
+## Alerts [/alerts/alert]
+
+### Get All Available Alerts [GET]
```

Comment:
For any endpoint that shows all entities (i.e. isnt accessed by its id) each entity should be expressed as an object using the singular of the pluralised top array key. 

The reason for this, is that our API automatically supports json/xml so we need to ensure it will have a valid node structure. 

In the .apib we do this by using a data structure wrapper pattern, in this instance, something like: 

```
## Alerts Wrapper (object)
+ alert (Alert Base)
```